lale/lib/rasl/min_max_scaler.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2021 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+
+import lale.docstrings
+import lale.operators
+from lale.expressions import it, max, min
+from lale.helpers import _is_spark_df
+from lale.lib.lale.aggregate import Aggregate
+from lale.lib.rasl import Map
+from lale.lib.sklearn import min_max_scaler
+
+
+class _MinMaxScalerImpl:
+    def __init__(self, feature_range=(0, 1), *, copy=True, clip=False):
+        self.feature_range = feature_range
+        self.copy = copy
+        self.clip = clip
```

Comment:
Maybe raise an exception if `clip` is true, so users don't get confused?

Reply:
See commit 1eb6090